github/repos.go

```diff
@@ -284,6 +287,7 @@ func (s *RepositoriesService) Get(owner, repo string) (*Repository, *Response, e
 	// TODO: remove custom Accept header when the license support fully launches
 	// https://developer.github.com/v3/licenses/#get-a-repositorys-license
 	req.Header.Set("Accept", mediaTypeLicensesPreview)
+	req.Header.Add("Accept", mediaTypeSquashPreview)
```

Comment:
> I will ask GitHub support and see what they recommend.

Good idea, thanks for doing that!

If it turns out that it's not possible to combine multiple beta APIs in same request, we might have to make 2 requests (each with a different beta API), or just pick 1 beta that's more important and remove the other. Or wait until at least one of them is merged into main v3 API, then use the remaining beta header.